types/src/verifier/mock.rs

```diff
@@ -5,7 +5,7 @@ use super::ProofVerifier;
 use crate::{PoStProof, RegisteredPoStProof, SealVerifyInfo, SectorInfo};
 use fvm_shared::randomness::Randomness;
 
-/// Mock verifier. This does no-op verification of any proofs.
+/// Mock `verifier`. This does no-op verification of any proofs.
```

Comment:
We might need to add this to the dictionary.

Reply:
Yeah I was on the fence about that one, I initially added it and then decided to remove it lol.